feat: first task

kgaleev opened the pull request:
* add time zone and language
* add version
* add /about with www.thinknetica.com
* add /about/author with github profile
* add project entities to README.md

kgaleev committed:
feat: first task
* add time zone and language
* add version
* add /about with www.thinknetica.com
* add /about/author with github profile
* add project entities to README.md

BubuntuClu commented on config/initializers/version.rb:
```diff
@@ -0,0 +1,3 @@
+module Version
+  VERSION = '1.0.0-alpha'
```
по документации стартовая версия приложения "0.1.0"

BubuntuClu commented on public/about.html:
```diff
@@ -0,0 +1,10 @@
+    <title>Intro to rails</title>
+</head>
+<body>
+site: www.thinknetica.com
```
это должна быть ссылка. и сторонние ресурсы открываем в новой закладке

BubuntuClu commented on public/about/author.html:
```diff
@@ -0,0 +1,10 @@
+    <title>About author</title>
+</head>
+<body>
+https://github.com/kgaleev
```
это должна быть ссылка. и сторонние ресурсы открываем в новой закладке

BubuntuClu commented on config/application.rb:
```diff
@@ -0,0 +1,24 @@
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module Intro
```
почему тут интро?

вообще этого файла целиком в ПРе не должно быть, только несколько новых строк

kgaleev replied:
название модуля не было регламентировано; думал может дальше будет прямо указано, если надо будет поменять

можно я не буду откатывать коммит, чтобы было только несколько новых строк? Это из-за того, что я все файлы руками перебирал. Не стал изменения откатывать уже, засунул целиком файл в коммит (иначе проще с нуля сделать проект)

BubuntuClu replied:
не понял причем тут откат коммита

kgaleev committed:
fix: changes for the pull request pull/1

- version is 0.1.0
- /about: make thinknetica.com an url, opens in a new tab
- /about/author: make github.com/kgaleev an url, opens in a new tab
- config/application.rb: rename module to TestGuru
